File: leetcode/max-consecutive-ones/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        counter = 0
        maxcounter = 0

        for num in nums:
            if num == 1:
                counter += 1
            else:
                counter = 0

            if counter > maxcounter:
                maxcounter = counter

        return maxcounter

File: leetcode/_ARRAYS_101/max-consecutive-ones/max-consecutive-ones.kt
class Solution {
    fun findMaxConsecutiveOnes(nums: IntArray): Int {
        var counter = 0
        var max = 0

        nums.forEach {
            if (it == 1) counter++
            else counter = 0

            if (counter > max) max = counter
        }

        return max
    }
}